docs/tutorials/configure/build-a-mock-robot.md

```diff
@@ -285,128 +302,99 @@ func main() {
+You will be creating a mock sub-part to control a motor.
+This could be anything, but imagine for the purpose of this tutorial that this motor controls a conveyor belt in front of your mock arm on an assembly line.
 
-Navigate to your new part's **config** page and create a new motor using the **fake** model.
-
-### How to add your sub-part as a remote
-
-Connecting your sub-part as a remote from your main robot will allow you to control your sub-parts all from one place inside of your main robot.
-
-From the **code sample** tab of your sub-part:
+On your robot's page on the Viam app, click on the dropdown next to the main part, name your part and click **Add new**.
 
-- Copy the **Config as Remote Part**.
-- Navigate back to the **config** and then the **Remotes** tab of your main robot
-- Paste your sub-part's configuration.
+![Screenshot of the Viam app with a dropdown below the main part. 'SubPart' is written in the textbox.](../../img/build-a-mock-robot/part-menu.png)
 
-<img src="../../img/build-a-mock-robot/image8.png" alt="Screenshot from the Viam app showing the CONFIG > REMOTES with the sub-part's remote config file pasted in." width="80%">
+Navigate to your new sub-part's **Config** tab and create a new motor using the **fake** model.
+Name it `"motor2"`.
 
-### How to start a new instance of `viam-server` for your mock sub-part
+![The config tab of the sub-part. A new motor component called motor2 is being created.](../../img/build-a-mock-robot/sub-part-motor.png)
 
-Since every part needs to run an instance of `viam-server`, you will need to bind the sub-part to a new port so we can run two servers on your machine at the same time.
-We are using port `8081`, but you can use any open port you want.
+Click **Create component** and then **Save config**.
 
-You can do this by going to **config** and then going to the **Auth/Network** tab.
-Here, you will paste the following:
+### Start a new instance of `viam-server` for your mock sub-part
 
-```json {class="line-numbers linkable-line-numbers"}
-{
-    "bind_address": "localhost:8081"
-}
-```
+Every part needs to run an instance of `viam-server`.
+Typically, a sub-part would represent a separate computer running `viam-server`.
+For this tutorial since you are using only one computer, you will need to bind the sub-part to a new port so you can run two servers on your machine at the same time.
+The following instructions use port `8081`, but you can use any open port you want.
 
-Be sure to save before continuing.
+1. Go to the **Config** tab and then go to the **Auth/Network** sub-tab.
+2. Under **Network** click **Add bind address**.
+3. In the **Host** field type `localhost`.
+4. In the **Port** field type `8081`.
+5. Click **Save config**.
 
-### How to run a second instance of `viam-server` for your sub-part
+### Run a second instance of `viam-server` for your sub-part
```

Comment:
need to point out that this happens on a third tab

Reply:
done